packages/protocol/contracts/common/GoldToken.sol

```diff
@@ -141,7 +141,11 @@ contract GoldToken is Initializable, CalledByVm, Freezable, IERC20, ICeloToken {
    * @param value The amount of cGLD to mint.
    */
   function mint(address to, uint256 value) external onlyVm returns (bool) {
-    require(value > 0, "mint value must be > 0");
+    require(value >= 0, "mint value must be >= 0");
```

Comment:
unnecessary check for unsigned int, please remove

Reply:
Ah, right!